src/apiCalls.js

```diff
@@ -18,18 +18,30 @@ export const fetchUsers = () => {
         }})
 }
 
-export const postFavorite = (user, movie) => {
-  const post = {
-      method: 'POST',
+export const addFavorite = (user, movie) => {
+  const patch = {
+      method: 'PATCH',
```

Comment:
LOVE the use of PATCH! I haven't used that at all so this is awesome to see implemented!